tw-tasks-core/src/main/java/com/transferwise/tasks/stucktasks/TasksResumer.java

```diff
@@ -83,6 +83,10 @@ public class TasksResumer implements ITasksResumer, GracefulShutdownStrategy, In
 
   @Override
   public void afterPropertiesSet() {
+    if (!tasksProperties.isAutoStartProcessing()) {
+      log.info("Task Resumer stopped.");
+      return;
+    }
```

Comment:
Same again think this check should go in the `applicationStarted` method

https://github.com/transferwise/tw-tasks-executor/blob/master/tw-tasks-core/src/main/java/com/transferwise/tasks/stucktasks/TasksResumer.java#L425-L429